hack/jenkins/linux_integration_tests_kvm.sh

```diff
@@ -29,7 +29,6 @@ OS="linux"
 ARCH="amd64"
 DRIVER="kvm2"
 JOB_NAME="KVM_Linux"
-EXPECTED_DEFAULT_DRIVER="kvm2"
```

Comment:
why do we delete this ?
on linux we still expect kvm to be default

Reply:
I removed it because we're not using it anywhere, those are the only usage of the value, we don't use it anywhere

Reply:
Also, these values are incorrect, the default driver would be Docker

Reply:
I see good call !